lib/core/src/Cardano/Wallet.hs

```diff
@@ -2239,6 +2244,7 @@ instance HasSeverityAnnotation WalletLog where
         MsgPaymentCoinSelectionStart{} -> Debug
         MsgPaymentCoinSelection _ -> Debug
         MsgPaymentCoinSelectionAdjusted _ -> Debug
+        MsgIsStakeKeyRegistered _ -> Info
```

Comment:
What about:

- `Info` when `False`
- `Debug` otherwise.

Otherwise.

Reply:
`False` would be expected for new wallets.

`True` would be common for people will ITN rewards.

Why not `Info` for both?

Reply:
It's true that we aren't going to see this log line often actually. Only occurs when changing delegation settings, so I guess it's fine :+1: